<h2 align="center" style="color:blue">Codebasics Python Course: Exercise - Variables, Numbers, Strings</h2>

### Task 1

Create a variable named pi and store the value 22/7 in it. Now check the data type of this variable.

In [1]:
# write your code here
pi = 22/7
pi

3.142857142857143

### Task 2

Create a variable called ```for``` and assign it a value 4. See what happens and find out the reason behind the behavior that you see.

In [2]:
# write your code here
for = 4

SyntaxError: invalid syntax (4203740755.py, line 2)

### Task 3

Below we have defined the variables for storing the principal amount, rate of interest and time. You need to calculate the simple interest for 3 years. Once simple interest is calculated, calculate the total amount you will have at the end of the tenure

In [4]:
principle_amount = 567.00
rate_of_interest = 5.6
time = 3

Now let's calculate the simple intrest using simple math formula:

`simple interest  = P x R x T / 100`

where: 
 
    P = principle amount
    R = rate of interest
    T = time 



In [5]:
# calculate simple interest
simple_interest = (principle_amount * rate_of_interest * time) / 100
simple_interest

95.25599999999999

In [6]:
# calculate total amount after including interest
total_amount = principle_amount + simple_interest
total_amount

662.256

### Task 4

There is a circular pond in a village. This pond has a radius of 84 meter. Can you find the area of the pond?

In [2]:
# write your code here
import math
radius = 84
pond_area = math.pi * (radius**2)
pond_area

22167.07776372958

### Task 5

If there is a 2000 liter water in a square meter, what is the total amount of water in this pond?

**Note:** For simplicity, use the value of pi as 3.14.

In [3]:
# write your code here
per_sqt_meter_water = 2000
total_water = pond_area*per_sqt_meter_water
total_water

44334155.52745916

### Task 6

If you cross a 490-meter-long street in 7 minutes, then what is your speed in meter per seconds. Print your answer with only two decimal points

Hint:
        Speed = Distance / Time

In [8]:
# write your code here
distance = 490
time = 7 * 60
speed = distance/time
round(speed,2)

1.17

### Task 7

Imagine you are a shop owner tracking sales of three products throughout the day. At the end of the day, you want to calculate the total sales from these products. Product quantity and prices are given below in the next code cell,

Task: Write a program that:

1. Calculates the total sales for each product.
1. Summarizes the total sales for the day.
1. Prints this summary in a formatted manner.

Expected Output:

```
Daily Sales Summary:
- Product 1: Sold 15 units at $20.0 each, Total: $300.0
- Product 2: Sold 10 units at $35.0 each, Total: $350.0
- Product 3: Sold 20 units at $12.0 each, Total: $240.0
Total Sales for the Day: $890.0
```

In [11]:
# Prices and quantities for each product
product1_price = 20.0  # Price per unit for product 1
product1_quantity = 15  # Quantity sold of product 1

product2_price = 35.0  # Price per unit for product 2
product2_quantity = 10  # Quantity sold of product 2

product3_price = 12.0  # Price per unit for product 3
product3_quantity = 20  # Quantity sold of product 3

In [13]:
# write your code here
product_1_sale = product1_price * product1_quantity
print(f"Product 1 sale is :{product_1_sale}")

product_2_sale = product2_price * product2_quantity
print(f"Product 1 sale is :{product_2_sale}")

product_3_sale = product3_price * product3_quantity
print(f"Product 1 sale is :{product_3_sale}")

total_sale = product_1_sale + product_2_sale + product_3_sale
print(f"Total sale :{total_sale}")

print("Daily Sales Summary:")
print(f"- Product 1: Sold {product1_quantity} units at ${product1_price} each, Total: ${product_1_sale}")
print(f"- Product 2: Sold {product2_quantity} units at ${product2_price} each, Total: ${product_2_sale}")
print(f"- Product 3: Sold {product3_quantity} units at ${product3_price} each, Total: ${product_3_sale}")
print(f"Total Sales for the Day: ${total_sale}")

Product 1 sale is :300.0
Product 1 sale is :350.0
Product 1 sale is :240.0
Total sale :890.0
Daily Sales Summary:
- Product 1: Sold 15 units at $20.0 each, Total: $300.0
- Product 2: Sold 10 units at $35.0 each, Total: $350.0
- Product 3: Sold 20 units at $12.0 each, Total: $240.0
Total Sales for the Day: $890.0
